src/scancode/cli.py

```diff
@@ -442,7 +443,14 @@ def scan_event(item):
 
     files_scanned_per_second = round(float(files_count) / scanning_time , 2)
     scan_summary['files_scanned_per_second'] = files_scanned_per_second
-
+    
+    if not scanning_errors:
+        #Log file containing timestamp & Project's name.
+        log = open (input_path + "_log.txt", "a+")
+        day = datetime.datetime.today().replace(microsecond=0)
```

Comment:
In `commoncode/timeutils.py`, there's a function `tstamp2time` which takes a UTC time stamp as input and converts to a datetime object & a function `time2tstamp` which returns a UTC time stamp. So, if we feed the latter to the former. It'll return a timestamp. 
In such a case, is it better to call `commoncode`?